crates/trie/sparse/src/parallel_trie.rs

```diff
@@ -47,3 +355,40 @@ pub struct SparseSubtrie {
     /// The map from paths to sparse trie nodes within this subtrie.
     nodes: HashMap<Nibbles, SparseNode>,
```

Comment:
I think we want to store `values` in this struct, because `rlp_node` will use it to calculate the hashes https://github.com/paradigmxyz/reth/blob/84e6668307b56fa9a1356dd48e646936fbb1a648/crates/trie/sparse/src/trie.rs#L1146